src/Common/ErrorHandlingUtils.ts

```diff
@@ -21,9 +21,15 @@ export const handleError = (error: string | ARMError | Error, area: string, cons
   sendNotificationForError(errorMessage, errorCode);
 };
 
-export const getErrorMessage = (error: string | Error): string => {
-  const errorMessage = typeof error === "string" ? error : error.message;
-  return replaceKnownError(errorMessage);
+export const getErrorMessage = (error: string | Error, includeStackTrace?: boolean): string => {
+  if (typeof error === "string") {
+    return replaceKnownError(error);
+  }
+
+  const errorMessage = replaceKnownError(error.message);
+  return includeStackTrace && error.stack
+    ? `Error message: ${errorMessage}\n Stack trace: ${error.stack}`
```

Comment:
can we make an object like 
error: {message: "....", stacktrace: "..."} 
in the telemetry message? Will be easier to parse out when we are looking at kusto logs.

Reply:
We want to avoid passing an object to the telemetry processor since it will get stringified. I think what Steve suggested below is a good alternative. I will add a new key for stack trace in traceFailure.